.github/workflows/actions.yml

```
@@ -0,0 +1,32 @@
+name: Continuous Integration
+on: [push]
+jobs:
+  main:
+    name: Specs
+    runs-on: [self-hosted, zendesk-stable]
+    env:
+      RAILS_ENV: test
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.6
+          - 2.7 # Default
+        gemfile:
+          - Gemfile # Default
+        task:
+          - spec
+    steps:
+    - uses: zendesk/checkout@v2
+    - uses: zendesk/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
```

Don't forget to install gems:
```suggestion
        ruby-version: ${{ matrix.ruby }}
        bundler-cache: true
```